slack_rules/slack_legal_hold_policy_modified.py

```diff
@@ -0,0 +1,26 @@
+from panther_base_helpers import deep_get, slack_alert_context
+
+LEGAL_HOLD_POLICY_ACTIONS = {
+    "legal_hold_policy_entities_deleted": "Slack Legal Hold Policy Entities Deleted",
+    "legal_hold_policy_exclusion_added": "Slack - Exclusions Added to Legal Hold Policy",
```

Comment:
the dash between slack and exclusions seems misplaced with none of the other texts having it.

Reply:
Nice catch :)